sdk/core/core-http/test/policies/bearerTokenAuthenticationPolicyTests.ts

```diff
@@ -73,6 +74,19 @@ describe("BearerTokenAuthenticationPolicy", function() {
     }
   });
 
+  it("tests that AccessTokenRefresher is working", async function() {
+    this.timeout(35000);
+    const now = Date.now();
+    const credentialToTest = new MockRefreshAzureCredential(now);
+
+    const request = createRequest();
+    const policy = createBearerTokenPolicy("testscope", credentialToTest);
+    await policy.sendRequest(request);
+    await delay(30000);
```

Comment:
You should stub this out like you did in the Identity tests so that we don't have to wait 30 seconds on a test run

Reply:
thank you! I'll do what you say

Reply:
I believe I did it here: https://github.com/Azure/azure-sdk-for-js/pull/10085/commits/7f3192ea69401bce8d2689e0ec4bdb66b5b991c1 thank you again for the feedback!